Resources/Prototypes/Entities/Clothing/OuterClothing/sinditrench.yaml

```
@@ -0,0 +1,20 @@
+- type: entity
```

```suggestion
- type: entity
```
Тоже самое. Фаил расширением .yaml